### PR TITLE
N44 update

### PR DESCRIPTION
**Pull request by marcelofcastro**
This pull request is made in order to merge N44 updates into the master. The changes in N44 are related to the addition of the models used for the paper in this [link](https://github.com/ALSETLab/2019_Modelica_Conf_DAESolvers4LargeHybridModels).

### CODE REVIEW

File: ApplicationExamples/N44/status. [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Simulating: Time= 0.512     
+
```
Comment (dietmarw) [inline ApplicationExamples/N44/status.]: This file should not be part of the commit.

File: ApplicationExamples/N44/DAEMode_TestCases/N44_Base_Case_Systems/Generators/GenEventTest.mo [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    S12=0.3,
+    R_a=0,
+    V_b=V_b,
+    V_0=V_0,
+    angle_0=angle_0,
+    P_0=P_0,
+    Q_0=Q_0) annotation (Placement(transformation(extent={{-64,-24},{-2,40}})));
+  OpenIPSL.Electrical.Controls.PSSE.TG.HYGOV hYGOV(
+    R=0.06 "Permanent droop, p.u",
+    r=0.4 "Temporary droop, p.u",
```
Comment (dietmarw) [inline ApplicationExamples/N44/DAEMode_TestCases/N44_Base_Case_Systems/Generators/GenEventTest.mo]: This is a strange place to add a description string. The description strings should be present in the original class not as part of the instance modifier.

File: ApplicationExamples/N44/DAEMode_TestCases/N44_Base_Case_Systems/Generators/GenEventTest.mo [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+          -1.6},{16,-55.3333},{22,-55.3333}},     color={0,0,127}));
+  connect(eXST1_1.VUEL, cte.y) annotation (Line(points={{32.15,-82},{-2,-82},
+          {-2,-64},{-9.4,-64}}, color={0,0,127}));
+  annotation (
+    Icon(graphics={Ellipse(extent={{-100,-100},{101,100}}, lineColor={0,0,255}),
+          Line(
+          points={{-76,-26},{-28,52},{27,-52},{74,23}},
+          color={0,0,255},
+          smooth=Smooth.Bezier)}),
+    Diagram(coordinateSystem(preserveAspectRatio=false, extent={{-100,-100},{
```
Comment (dietmarw) [inline ApplicationExamples/N44/DAEMode_TestCases/N44_Base_Case_Systems/Generators/GenEventTest.mo]: pointless Diagram and Documentation annotations that should be removed.

File: ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Generator/GenEventTest.mo [hunk 4]
```diff
@@ -0,0 +1,98 @@
+within N44.DAEMode_TestCases.SMIB_Examples.Example_1.Generator;
+model GenEventTest "Configuration of synchronous generator with regulators: GENSAL, HYGOV, SCRX, STAB2A;
+  Nordic 44 model: Buses 3115, 5300, 6100, 6700, 7100"
+    extends OpenIPSL.Electrical.Essentials.pfComponent;
+    OpenIPSL.Electrical.Machines.PSSE.GENSAL gENSAL(
+          Tppd0=0.05 "T''do (> 0)",
```
Comment (dietmarw) [inline ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Generator/GenEventTest.mo]: Another example of description strings in the wrong place.

File: ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Generator/GenEventTest.mo [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+          R_a=0,
+          V_b=V_b,
+          V_0=V_0,
+          angle_0=angle_0,
+          P_0=P_0,
+          Q_0=Q_0) annotation (Placement(transformation(extent={{-64,-24},{-2,40}})));
+    OpenIPSL.Electrical.Controls.PSSE.TG.HYGOV hYGOV(
+          R=0.06 "Permanent droop, p.u",
+          r=0.4 "Temporary droop, p.u",
+          VELM=0.2 "Gate open/close velosiy limit, p.u/sec",
```
Comment (dietmarw) [inline ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Generator/GenEventTest.mo]: per-unit unit is either "pu" which I would prefer, or "p.u." but no mix.

File: ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Generator/GenEventTest.mo [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+    connect(eXST1_1.VUEL, cte.y) annotation (Line(points={{34.15,-82},{-2,
+                -82},{-2,-64},{-9.4,-64}},
+                                      color={0,0,127}));
+    annotation (
+          Icon(graphics={Ellipse(extent={{-100,-100},{101,100}}, lineColor={0,0,255}),
+                Line(
+                points={{-76,-26},{-28,52},{27,-52},{74,23}},
+                color={0,0,255},
+                smooth=Smooth.Bezier)}),
+          Diagram(coordinateSystem(preserveAspectRatio=false, extent={{-100,-100},{
```
Comment (dietmarw) [inline ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Generator/GenEventTest.mo]: remove empty / default annotations.

File: ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Network1.mo [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+        Line(points={{-51,0},{-60,0},{-60,0},{-60,0}},          color = {0, 0, 255}));
+      connect(G1.pwPin, B1.p) annotation (
+        Line(points={{-77,0},{-62,0},{-62,0},{-60,0}},          color = {0, 0, 255}));
+        annotation (
+      experiment(
+        StartTime=0,
+        StopTime=10,
+        Tolerance=1e-6,
+        Interval=1e-4),
+      Documentation);
```
Comment (dietmarw) [inline ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Network1.mo]: empty annotation

File: ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Network2.mo [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+        Line(points={{-14,-20},{-26,-20},{-26,0},{-40,0},{-40,0}},            color = {0, 0, 255}));
+      connect(twoWindingTransformer.n, B2.p) annotation (
+        Line(points={{-49,0},{-41,0},{-41,0},{-40,0}},          color = {0, 0, 255}));
+      connect(G1.pwPin, B1.p) annotation (
+        Line(points={{-87,0},{-80,0}},      color = {0, 0, 255}));
+      connect(twoWindingTransformer.p, B1.p) annotation (
+        Line(points={{-71,0},{-80,0}},      color = {0, 0, 255}));
+      annotation (
+        experiment(StartTime = 0, StopTime = 10, Tolerance = 1e-6, Interval = 1e-4),
+        Documentation,
```
Comment (dietmarw) [inline ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Network2.mo]: more empty annotations

File: ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Network3.mo [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+        Line(points={{-51,0},{-60,0},{-60,0},{-60,0}},          color = {0, 0, 255}));
+      connect(G1.p, B1.p)
+        annotation (Line(points={{-77,0},{-60,0}}, color={0,0,255}));
+        annotation (
+      experiment(
+        StartTime=0,
+        StopTime=10,
+        Tolerance=1e-6,
+        Interval=1e-4),
+      Documentation);
```
Comment (dietmarw) [inline ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Network3.mo]: empty annotation

File: ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Network1.mo (hunk 7 above)
Comment (marcelofcastro) [reply ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Network1.mo]: @dietmarw do you want me to erase the part of the documentation with the experiment settings?

Comment (dietmarw) [reply ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Network1.mo]: yes. Please.

Comment (marcelofcastro) [reply ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Network1.mo]: @dietmarw Ok, I see. But since these are not default values, and the simulated system is an example, I thought that experiment settings would allow the user to reach the expected results. Don't you think so?

**Comment by dietmarw** [comment]
You misunderstood. Leave the experiment annotation as it is but remove the
documentation annotation which is empty and useless here.

File: ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Network1.mo (hunk 7 above)
Comment (dietmarw) [reply ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Network1.mo]: So basically only remove 
",
Documentation"

**Comment by dietmarw** [comment]
@marcelofcastro If you want I can run a clean up script https://github.com/dietmarw/trimtrailingwhitespaces push to your PR and then we take it from there.

File: ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Network1.mo (hunk 7 above)
Comment (marcelofcastro) [reply ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Network1.mo]: Oh ok. Yes, I misunderstood that. Thanks!

**Comment by marcelofcastro** [comment]
@dietmarw that's ok, don't need to run the script this time.

**Comment by dietmarw** [comment]
I guess I will do it anyway before merge so we start off with a nice and clean N44 package.

**Comment by dietmarw** [comment]
But I'll wait until all other issues are resolved to not cause any unnecessary merge conflicts.

**Comment by marcelofcastro** [comment]
Yeah, that sounds good!

File: ApplicationExamples/N44/DAEMode_TestCases/N44_Base_Case_Systems/Generators/GenEventTest.mo (hunk 3 above)
Comment (dietmarw) [reply ApplicationExamples/N44/DAEMode_TestCases/N44_Base_Case_Systems/Generators/GenEventTest.mo]: my script will take care of those.

**Comment by marcelofcastro** [comment]
I think I solved most of the empty documentation, and I guess I did solve the description of variables in the extensions. Gonna, check it again.

**Comment by dietmarw** [comment]
The former my script would have taken care of, it is the latter that needs manual work ;-)

**Comment by marcelofcastro** [comment]
Yep, I know.

File: ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Generator/GenEventTest.mo (hunk 5 above)
Comment (marcelofcastro) [reply ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Generator/GenEventTest.mo]: Since it is also about descriptions in modifiers of a model, I rather delete it instead of correcting the term per unit.

Comment (dietmarw) [reply ApplicationExamples/N44/DAEMode_TestCases/SMIB_Examples/Example_1/Generator/GenEventTest.mo]: Yes sure. No point in correcting here.

**Comment by dietmarw** [comment]
Now that all annotations are cleaned up we need to rebase this PR onto the latest state of master. I already know that this will break since some changes were done during PR #157 that are not included here.

**Comment by dietmarw** [comment]
@marcelofcastro rebase is done and I fixed the (only) issue with XADIFD connections (which I already knew about). Since I needed to force push, you need to reset your branch to the new head of this `N44_Update ` branch. 

There are still some things left. This application example does actually follow the one-file per model structure. This should be fixed before merging this finally.

**Comment by dietmarw** [comment]
Re-save is now done an all models check. @marcelofcastro You can have a final look and then feel free to merge.
